### DISCUSSION TITLE
Lambda Deployment

### DISCUSSION THREAD

**Discussion by yousufAtBazaar** [aminyazdanpanah/python-ffmpeg-video-streaming#116] [Q&A]
I am getting following error when deployed to lambda

> {
  "errorMessage": "cannot pickle 'SSLContext' object",
  "errorType": "TypeError",
  "stackTrace": [
    "  File \"/var/task/lambda_function.py\", line 36, in lambda_handler\n    hls = video.hls(Formats.h264())\n",
    "  File \"/opt/python/ffmpeg_streaming/_input.py\", line 139, in method\n    media = Media(self)\n",
    "  File \"/opt/python/ffmpeg_streaming/_media.py\", line 246, in __init__\n    first_input = dict(copy.deepcopy(_inputs.inputs[0]))\n",
 ..............
  ]
}

here is the code

```
def lambda_handler(event, context):
    s3 = S3()
    _480p = Representation(Size(480,854), Bitrate(750 * 1024, 192 * 1024), hls_time=2, preset="veryfast")
    _720p = Representation(Size(720,1280), Bitrate(2048 * 1024, 320 * 1024), hls_time=2, preset="veryfast")
    _1080p = Representation(Size(1080,1920), Bitrate(4096 * 1024, 320 * 1024), hls_time=2, preset="veryfast")

    video = ffmpeg_streaming.input(s3, bucket_name="bucket_name",
                                   key="path/to/file.mp4")

    save_to_s3 = CloudManager().add(s3, bucket_name="bucket_name",
                                    folder=f'path/to/output_folder')

    hls = video.hls(Formats.h264())
    hls.representations(_480p, _720p, _1080p)
    hls.output(clouds=save_to_s3, monitor=monitor)
```
Any kind of help would be appreciated. thanks.

**Comment by gkrwns7566** [comment]
Upgrading to the latest version might solve your problem
